packages/actor-http-memento/package.json

```diff
@@ -40,8 +40,10 @@
     "@comunica/bus-http": "^3.2.0",
     "@comunica/context-entries": "^3.2.0",
     "@comunica/core": "^3.2.0",
-    "@types/parse-link-header": "^2.0.0",
     "cross-fetch": "^4.0.0",
-    "parse-link-header": "^2.0.0"
+    "http-link-header": "^1.1.3"
+  },
+  "devDependencies": {
+    "@types/http-link-header": "^1.0.7"
```

Comment:
This can be an actual dep.

Reply:
I don't believe that is necessary in this case none of the types from `@types/http-link-header` are re-exported.

Reply:
AFAIK, all of our types packages are non-dev deps (some probably also don't re-export types like here), so it would be good to remain consistent with the rest of the codebase.
Also, if this actor would somehow expose `http-link-header` types in the future, we may introduce a bug if we're not careful.
These packages are tiny anyways, and do not increase bundle size.